plugins/generator-1.18.2/forge-1.18.2/procedures/entity_has_screen_open.java.ftl

```diff
@@ -0,0 +1 @@
+${input$entity} instanceof Player player ? player.containerMenu instanceof ${field$guiname}Menu : false
```

Comment:
The template should be surrounded by parentheses. The pattern variable should be renamed to `_plr`, to avoid collisions with other variables (Same for 1.19.2)